docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -0,0 +1,459 @@
+Using Public Clouds as backends
+===============================
+
+Introduction
+------------
+
+As stated in our `GETTING STARTED guide <../GETTING_STARTED/#location-configuration>`__,
+new backends can be added by creating a region (also called location constraint)
+with the right endpoint and credentials.
+This section of the documentation shows you how to set up our currently
+supported public cloud backends:
+
+- `Amazon S3 <#aws-s3-as-a-backend>`__ ;
+- `Microsoft Azure <#microsoft-azure-as-a-backend>`__ .
+
+For all Public Cloud backends, you'll have to do some editing inside the
+repository, and some editing from the Public Cloud Console.
+
+.. IMPORTANT::
+   CloudServer also enables you to have, within the same local bucket, objects
+   stored to different backends (both private and public clouds, for example).
+   This is an advanced usecase, and it will require that you set your backends
+   as explained below. Once this is done, you may refer to the
+   `backend configuration at object level <#backend-configuration-at-object-level>`__
+   section of this document.
+
+AWS S3 as a backend
+-------------------
+
+From the AWS S3 Console
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a bucket where you will host your data for this new location constraint.
+This bucket must have versioning enabled. This is an option you may choose to
+activate at step 2 of Bucket Creation in the console.
+In this example, our bucket will be name ``zenkobucket`` and has versioning
+enabled.
+
+From the CloudServer repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+locationConfig.json
+^^^^^^^^^^^^^^^^^^^
+
+Edit this file to add a new region, which should describe the AWS S3 hosted
+bucket you'll be writing your data to. There are a few configurable options here:
+
+- :code:`type` : set to :code:`aws_s3` to indicate this region is hosted in AWS S3;
+- :code:`legacyAwsBehavior` : set to :code:`true` to indicate this region should
+  behave like AWS S3 :code:`us-east-1` region, set to :code:`false` to indicate
+  this region should behave like any other AWS S3 region;
+- :code:`bucketName` : set to an *existing bucket* in your AWS S3 Account; this
+  is the bucket that will host all the data written in this region;
```

Comment:
this is the bucket in which your data will be stored for this location constraint

Reply:
changed